### PR TITLE
cmd/atlas: support --revisions-schema in project file

### PR DESCRIPTION
**Pull request by ronenlu**

### CODE REVIEW

File: cmd/atlas/internal/cmdapi/project.go [hunk 1]
```diff
@@ -73,6 +73,9 @@ type Env struct {
 	// List of schemas in this database that are managed by Atlas.
 	Schemas []string `spec:"schemas"`
 
+	// schema name where the revisions' table resides (default "atlas_schema_revisions").
```
Comment (giautm) [inline cmd/atlas/internal/cmdapi/project.go:76]:
```suggestion
	// schema name where the revisions's table resides (default "atlas_schema_revisions").
```

**Review by giautm — COMMENTED** [review]
LGMT,

File: cmd/atlas/internal/cmdapi/project_test.go [hunk 2]
```diff
@@ -27,6 +27,7 @@ env "local" {
 	dev = "docker://mysql/8"
 	src = "./app.hcl"
 	schemas = ["hello", "world"]
+	revisions_schema = "revisions"
```
Comment (a8m) [inline cmd/atlas/internal/cmdapi/project_test.go:30]: Looks good, but I'd love to merge it with `migration` in next PRs:
```hcl
migration {
    url = "file://migrations"
    revisions_schema = "revisions"
}
```

Comment (a8m) [reply cmd/atlas/internal/cmdapi/project_test.go:30]: cc @rotemtam

Comment (ronenlu) [reply cmd/atlas/internal/cmdapi/project_test.go:30]: what do you mean by that?

**Review by a8m — APPROVED** [review]

**Pull request merged**